doc/radosgw/qat-accel.rst

```diff
@@ -142,14 +167,65 @@ Configuration
 
       export QAT_SECTION_NAME=CEPH
 
-   Next, edit the Ceph configuration file to enable QAT support for compression::
+   **For in-tree QAT**
+
+   For Intree QAT, Make sure your system meets `QATlib System Requirements`_ .
+   And QATlib can be installed from pre-built packages or source code,
+   please see `QATlib Installation`_ . After ready the qatlib, you can run
+   ``cpa_sample_code`` to check if the QAT environment is OK.
+
+   If you using QATlib source code, Ceph cmake build has enabled the compilation options of
+   qatlib and qatzip by default. Our normal compilation already includes QAT compressor related code.
+
+   .. prompt:: bash $
+
+      ./do_cmake.sh
+
+   Or if you using pre-built packages installed on system, the following CMake options have to be configured
+   when building Ceph:
+
+   .. prompt:: bash $
+
+      ./do_cmake.sh -DWITH_SYSTEM_QATLIB=ON -DWITH_SYSTEM_QATZIP=ON
+
+
+   **For both out-of-tree QAT and in-tree QAT**
+
+   Next, edit the Ceph configuration file to enable QAT support for *zlib* compression::
 
       qat compressor enabled=true
 
 
+   Set the RGW compression method:
+
+   .. prompt:: bash $
+
+      # for storage class(STANDARD)
+      radosgw-admin zone placement modify --rgw-zone=default --placement-id=default-placement --compression=zlib
+      # or create a new storage class(COLD) and define data pool(default.rgw.cold.data)
+      radosgw-admin zonegroup placement add --rgw-zonegroup default --placement-id default-placement --storage-class COLD
+      radosgw-admin zone placement add --rgw-zone default --placement-id default-placement --storage-class COLD --compression zlib --data-pool default.rgw.cold.data
+
+CONFIG REFERENCE
+================
+The following QAT related settings can be added to the Ceph configuration file
+(i.e., usually `ceph.conf`) under the ``[client.rgw.{instance-name}]`` section.
+
+.. confval:: qat_compressor_session_max_number
+.. confval:: qat_compressor_data_fmt
+.. confval:: qat_compressor_busy_polling
```

Comment:
the docs build is failing because these options don't exist yet (they're added in https://github.com/ceph/ceph/pull/58046)

Reply:
@cbodley, I'll wait for that to land, and then I'll merge and backport.

Reply:
thanks @zdover23. https://github.com/ceph/ceph/pull/58046 just merged. its squid backport is pending in https://github.com/ceph/ceph/pull/58643